src/api/gnosisProtocol/legacy/types.ts

```diff
@@ -35,7 +35,7 @@ export type LegacyFeeQuoteParams = FeeQuoteParams & {
   fromDecimals: number
   toDecimals: number
   chainId: ChainId
-  priceQuality?: string
+  priceQuality: string
```

Comment:
for convenience I prefer to be explicit. The quote logic is a bit all over the place, its easy to forget in one point to provide the correct value here. So I prefer to not rely on "defaults" 

Making it mandatory, it will make us decide which priceQuality we need in each moment

Reply:
Now i can be more specific 6d61111fc0f0e7696e19b53418a3a7b0653c52ae